package.json

```diff
@@ -100,8 +111,8 @@
     "react-router-dom": "4.4.0-beta.6"
   },
   "engines": {
-    "node": ">=8.9.4",
+    "node": ">=8.9",
     "npm": ">=5",
-    "yarn": "~1.3.0||>=1.5.0"
+    "yarn": ">=1.13"
```

Comment:
It is interesting how one could argue this being a breaking change. 😀

Reply:
Hmm why would you say it's a breaking change? You mean for contributors?

Reply:
Yeah. Anybody installing, not having that version, will at least get a warning. I am not saying it should but it is an interesting thought.

Reply:
Yeah, we can also look it the other way around: local and CI should use the same yarn version to avoid possible weird issues 😜